src/sentry/api/endpoints/warmup.py

```diff
@@ -0,0 +1,26 @@
+import django.urls
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import Endpoint, all_silo_endpoint
+from sentry.ratelimits.config import RateLimitConfig
+
+
+@all_silo_endpoint
+class WarmupEndpoint(Endpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+    owner = ApiOwner.UNOWNED
+    permission_classes = ()
+    rate_limits = RateLimitConfig(group="INTERNAL")
+
+    def get(self, request: Request) -> Response:
+        # load up the django url resolver
+        resolver = django.urls.get_resolver()
+        if not getattr(resolver, "_populated", False):
+            resolver._populate()
```

Comment:
this is unlikely to make any difference whatsoever -- the populated data is in a thread local so this will only warm 1/16 of the threads

also I would remove the getattr guard -- it _should_ error if this is wrong rather than silently continuing

Reply:
got it. yeah i was encountering a type error, i assume because a private var isn't included in django stubs. i'll change this.

and good point re: the thread local. will try something different.

Reply:
that's probably a good indication we shouldn't be mucking with private implementation details either :)

Reply:
- excluded the url from sentry APM and access logs
- changed wsgi.py to call `reverse`, which will populate the url resolver instead of having to do anything inside the endpoint.
- didn't see any errant logs resulting from this on my machine